samples/bot-conversation/nodejs/bots/teamsConversationBot.js

```diff
@@ -264,9 +272,10 @@ class TeamsConversationBot extends TeamsActivityHandler {
             );
 
             const convoParams = {
-                members: [member],
-                tenantId: context.activity.channelData.tenant.id,
-                activity: context.activity
+                members: [isAadid?{ id: member.aadObjectId } : { id: member.id}],
```

Comment:
nit: spacing should be [isAadid ? { id: member.aadObjectId } : { id: member.id }]

Reply:
Fixed